src/Components/WebView/Platforms/WebView2/src/WebView2WebViewManager.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Web.WebView2.Core;
+
+namespace Microsoft.AspNetCore.Components.WebView.WebView2
+{
+    public class WebView2WebViewManager : WebViewManager
```

Comment:
At some point we'll need to decide whether we want all this stuff to be public. It's not really an interesting part of the API surface from a public consumption perspective, yet it allows us to have common code between WPF, WinForms, and .NET MAUI.

No need to decide anything now. Keeping this in a separate assembly will help us ensure we have good layering for now.

Reply:
I agree, the layering is fine for now.

Later on we should consider replacing the M.A.C.WebView.WebView2 package with use of a shared-source file for the `WebView2WebViewManager` abstraction. It would make no difference to functionality, but would mean we just don't have to ship an extra publicly-consumable package for what is intended as an internal abstraction.